spec/features/login_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe "Login" do
+  scenario "with cis2 feature flag enabled" do
+    given_the_cis2_feature_flag_is_enabled
+    when_i_go_to_the_service
+    then_i_should_see_the_cis2_login_button
+  end
+
+  scenario "with cis2 feature flag disabled" do
+    given_the_cis2_feature_flag_is_disabled
+    when_i_go_to_the_service
+    then_i_should_see_the_start_now_button
+    and_i_should_not_see_the_cis2_login_button
+  end
+
+  def given_the_cis2_feature_flag_is_enabled
+    Flipper.enable(:cis2)
+  end
+
+  def given_the_cis2_feature_flag_is_disabled
+    Flipper.disable(:cis2)
+  end
+
+  def when_i_go_to_the_service
+    visit "/"
+  end
+
+  def then_i_should_see_the_cis2_login_button
+    expect(page).to have_button "Care Identity"
```

Comment:
```suggestion
    expect(page).to have_button "Log in with my Care Identity"
```

Should we check for this to ensure the content from the CSS is coming through as well?

Reply:
Ugh. That's right, I did try this, and it did blow up, and I did ¯\\\_(ツ)_/¯ and did move on. _le sigh_

![image](https://github.com/nhsuk/manage-vaccinations-in-schools/assets/15608/f45b1faa-3c85-49b2-80dc-2765e34f77be)

I'm not sure this is worth the effort to figure out, but if you know of a way to make this work happy to implement.